fixtures/u2_func_simple.rb.sem.uast

```diff
@@ -0,0 +1,159 @@
+{ '@type': "module",
+   '@role': [Identifier, Module, Statement],
+   file: { '@type': "uast:FunctionGroup",
+      '@pos': { '@type': "uast:Positions",
+         start: { '@type': "uast:Position",
+            offset: 0,
+            line: 1,
+            col: 1,
+         },
+         end: { '@type': "uast:Position",
+            offset: 2,
+            line: 1,
+            col: 3,
+         },
+      },
+      Nodes: [
+         { '@type': "uast:Alias",
+            Name: { '@type': "uast:Identifier",
+               Name: "testfnc1",
+            },
+            Node: { '@type': "uast:Function",
+               Body: { '@type': "uast:Block",
+                  Statements: [
+                     { '@type': "begin",
```

Comment:
`begin` should be a `uast:Block` in fact

Reply:
True, fixed.